networks/typical-besu/variables.tf

```diff
@@ -45,7 +45,7 @@ variable "tessera_docker_image" {
 
 variable "ethsigner_docker_image" {
   type        = object({ name = string, local = bool })
-  default     = { name = "consensys/quorum-ethsigner:develop", local = false }
+  default     = { name = "consensys/ethsigner:latest", local = false }
```

Comment:
Does the EthSigner image in docker-help/variables.tf also need updating? It's also currently using `consensys/quorum-ethsigner:develop`

Reply:
yep fixed. there was another spot too - typical-hybrid was already using latest but I updated the repo